packages/forms-web-app/src/dynamic-forms/dynamic-components/radio/index.njk

```diff
@@ -41,6 +41,10 @@
           attributes: { "data-cy":"button-save-and-continue"}
         }) }}
       </form>
+
+      <p class="govuk-body">
+        <a href="{{listLink}}" class="govuk-link">Return to questionnaire homepage</a>
```

Comment:
Should the return link not be on the layout file rather than a specific individual component?

Reply:
I was worried that not every question would have it but probably makes more sense to put in template for now and figure out what to do if that comes up

Reply:
It looks like the link still needs to be removed from the radio component.

Reply:
apologies